src/index.ts

```diff
@@ -117,11 +129,21 @@ class ExpoIntegration {
   }
 }
 
-const originalSentryInit = Sentry.init;
-export const init = (options = {}) => {
-  options.integrations = [
-    ...(options.integrations || []),
-    new Sentry.Integrations.ReactNativeErrorHandlers({
+export const init = (options: ExpoNativeOptions | ExpoWebOptions = {}) => {
+  if (Platform.OS === 'web') {
+    return initBrowser({
+      ...(options as ExpoWebOptions),
+      enabled: __DEV__ ? options.enableInExpoDevelopment ?? false : true,
+    });
+  }
+
+  let optionsCopy = { ...options } as ExpoNativeOptions;
+
+  optionsCopy.integrations = [
```

Comment:
can we rename this to `nativeOptions`? I added a commit that did this but I think your force-push overwrote it/subsequent commits 🙈